packages/cx-elements/src/components/Spacing/index.static.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
I don't think we need static versions of element tokens.  I removed them from the token shdow pr.